src/services/uploads/src/constants.ts

```diff
@@ -39,6 +39,10 @@ export const STATUS_ERROR_PROCESSING_FILE: string =
   process.env.STATUS_ERROR_PROCESSING_FILE || "ERROR";
 export const STATUS_SKIPPED_FILE: string =
   process.env.STATUS_SKIPPED_FILE || "SKIPPED";
+export const STATUS_EXTENSION_MISMATCH_FILE: string = 
+  process.env.STATUS_EXTENSION_MISMATCH_FILE || "EXTMISMATCH"
+export const STATUS_UNKNOWN_EXTENSION: string = 
```

Comment:
Nitpicking here.  For naming consistency:
```
STATUS_EXTENSION_MISMATCH_FILE
STATUS_EXTENSION_UNKNOWN
```
But that'll also mean changing the env var name too.  Completely up to you.   
Aside from that, everything looks great